.circleci/config.yml

```diff
@@ -38,9 +38,15 @@ jobs:
     docker:
       - image: ornladios/adios2-auto:fedora-openmpi-ubsan
 
+  "fedora-asan":
+    <<: *defaults
+    docker:
+      - image: kylefromkitware/adios2-auto:fedora-asan
```

Comment:
This is a temporary image I've been using for development. Could someone build this image and upload it to `ornladios`, or add me to the organization so I can do it myself?